### PR TITLE
fix(color-field): added missing dependencies

### PR DESCRIPTION
**Pull request by Rajdeepc**
## Description
Added missing dependencies in Color-Field
```
@ctrl/tinycolor
@spectrum-web-components/textfield
```
Also added `@spectrum-web-components/color-field` in `@spectrum-web-components/bundle`

## Related issue(s)

- https://github.com/adobe/spectrum-web-components/issues/4140

## How has this been tested?

-   [ ] _Test case 1_
    1. Go here
    2. Do this
-   [ ] _Test case 2_
    1. Go here
    2. Do this

## Types of changes

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [ ] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: packages/color-field/package.json [hunk 1]
```diff
@@ -56,7 +56,9 @@
         "lit-html"
     ],
     "dependencies": {
-        "@spectrum-web-components/base": "^0.41.2"
+        "@ctrl/tinycolor": "^4.0.3",
+        "@spectrum-web-components/base": "^0.41.2",
+        "@spectrum-web-components/textfield": "^0.41.2"
```
Comment (Westbrook) [inline packages/color-field/package.json:61]: I think we also need `@spectrum-web-components/color-handle`.

**Comment by Westbrook** [comment]
The tachometer tests in https://github.com/adobe/spectrum-web-components/actions/runs/8172911409?pr=4141 are failing here in a way that I don't expect. Can you look into that? If you run them locally, and they still fail, opening the test link while it's attempting to run will allow you to see the error it encounters in the browser.

**Comment by Rajdeepc** [comment]
> The tachometer tests in https://github.com/adobe/spectrum-web-components/actions/runs/8172911409?pr=4141 are failing here in a way that I don't expect. Can you look into that? If you run them locally, and they still fail, opening the test link while it's attempting to run will allow you to see the error it encounters in the browser.

Checked this and it is failing in local also with the below error clicking the test link. This is referring that it is not able to find the correct reference of `bench-runner.html:1 Uncaught (in promise) TypeError: Failed to resolve module specifier "@spectrum-web-components/color-field/sp-color-field.js". Relative references must start with either "/", "./", or "../".`

**Comment by Westbrook** [comment]
That likely means that we didn't get Color Field added to `@spectrum-web-components/bundle` appropriately. Can you check on including that in this PR?

**Comment by Rajdeepc** [comment]
> That likely means that we didn't get Color Field added to `@spectrum-web-components/bundle` appropriately. Can you check on including that in this PR?

I could see `'@spectrum-web-components/color-field'` in `bundle/index.ts` and `elements.ts` but it was missing as a dependency. 
Added here `bundle/package.json` in its `dependency` object

**Review by Westbrook — COMMENTED** [review]
Nice! Thanks for cleaning this up. :shipit:

**Review by Westbrook — APPROVED** [review]

**Pull request merged**